src/frontegg-auth/src/auth.rs

```diff
@@ -250,6 +278,14 @@ impl Authenticator {
                         .with_label_values(&["new"])
                         .inc();
 
+                    // Make sure there is always something driving the request to completion
+                    // incase the client goes away.
+                    let request_ = request.clone();
+                    mz_ore::task::spawn(|| "auth-session-listener", async move {
+                        // We don't care about the result here, someone else handles it.
+                        let _ = request_.await;
+                    });
```

Comment:
nit: I like to introduce new scopes when I need clones for a closure or async block, to avoid ever having to juggle multiple variable names.

```suggestion
                    mz_ore::task::spawn(|| "auth-session-listener", {
                        let request = request.clone();
                        async move {
                            // We don't care about the result here, someone else handles it.
                            let _ = request.await;
                        }
                    });
```

Reply:
Ah ha, I like this pattern!